massgen/frontend/displays/textual_terminal_display.py

```
@@ -5013,6 +5014,21 @@ def _handle_coordination_event_side_effects(self, event) -> None:
+                try:
+                    evo_num = event.data.get("evolution_number", "?")
+                    evolved_count = event.data.get("evolved_count", 0)
+                    total_count = event.data.get("total_count", 0)
+                    summary = event.data.get("summary", "")
+                    short_summary = (summary[:80] + "...") if len(summary) > 80 else summary
+                    self.notify(
+                        f"Criteria evolved (v{evo_num}): " f"{evolved_count}/{total_count} criteria raised\n" f"{short_summary}",
+                        severity="information",
+                        timeout=10,
+                    )
+                except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)